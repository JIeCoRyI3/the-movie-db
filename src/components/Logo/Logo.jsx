import React from 'react';
import { footerLogo } from './Logo.module.css';
import logo from '../../assets/images/logo.png';

export default function Logo() {
    return (
        <a href="/">
            <img className={footerLogo} src={logo} alt="logo" />
        </a>
    );
}
