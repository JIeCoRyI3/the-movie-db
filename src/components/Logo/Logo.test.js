import React from 'react';
import { shallow } from 'enzyme';
import Logo from './index';

describe('<Footer />', () => {
    const wrapper = shallow(<Logo />);

    it('should be rendered', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
