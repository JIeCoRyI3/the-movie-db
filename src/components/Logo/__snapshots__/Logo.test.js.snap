// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should be rendered 1`] = `
<img
  alt="logo"
  src="test-file-stub"
/>
`;
